samples/README.md

```diff
@@ -0,0 +1,34 @@
+CORS Sample
+===
+This sample consists of a request origin (SampleOrigin) and a request destination (SampleDestination). 
+Both have different domain names, to simulate a CORS request. 
+
+Modify Hosts File
+Windows:
```

Comment:
This Markdown also won't render quite right: Markdown generally ignores single newlines. Recommend adding some additional formatting, such as bullet-points, section headings, etc. to get it to format correctly.

This section should probably also say *why* you have to modify the hosts file. E.g. "To run this CORS sample, modify the `hosts` file to register the hostnames:"